packages/tables/frontend/sirius-components-tables/src/graphql/mutation/useTableMutation.ts

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { useMutation } from '@apollo/client';
+import {
+  UseTableMutationValue,
+  GQLResizeColumnInput,
+  GQLResizeColumnData,
+  GQLResizeColumnVariables,
+} from './useTableMutation.types';
+import { useReporting } from '@eclipse-sirius/sirius-components-core';
+import { resizeColumnMutation } from './tableMutation';
+
+export const useTableMutations = (
```

Comment:
Please don't start creating utility files to put barely related stuff together. It creates tons of needless coupling in the end.